### ISSUE TITLE
Azure Data Factory Delete Items removes Linked Services

### ISSUE THREAD

**Issue by kvnkho** [liprec/vsts-publish-adf#53]
Hi,

I used the Azure Data Factory Delete Items task with Azure Devops to copy an Azure Data Factory from one environment to another. My intention is to delete triggers but not linked services. Below is the YAML configuration of the task:

```
steps:
- task: liprec.vsts-publish-adf.delete-adf-items.delete-adf-items@2
  displayName: 'Delete items from xxxxxxxxxxx'
  inputs:
    azureSubscription: 'XXXXXXXXX'
    ResourceGroupName: XXXXXXXXX
    DatafactoryName: XXXXXXXXX
    TriggerFilter: '*'

```

Even though I left LinkedServices blank, it seems to be deleting all of them. When I provide a value to the LinkedServices as the following:

`    ServiceFilter: TESTINGTESTING`

it works as expected and does not delete the said LinkedServices

My Task Version is 2.* (can't find if 2.0 or 2.2)

Thanks for the help!

**Comment by liprec** [comment]
HI @kvnkho,

If there is no ServiceFilter provided, the task should not delete the linked service.
Can you share the logging of a pipeline run with the variable `system.debug` set to `true`.

**Comment by kvnkho** [comment]
Here is my log. I went through it. ServiceFilter is undefined but LinkedServices are being deleted. 

[tasklog_7.log](https://github.com/liprec/vsts-publish-adf/files/4410070/tasklog_7.log)

Blurred out subscription id and subscription name just to be safe.

Thanks for looking at it!

**Comment by liprec** [comment]
Sorry for the late response, but I can confirm that this is a bug and it will be fixed in the next release.

**Comment by liprec** [comment]
If fixed in the latest version 2.2.2 of the delete task

**Comment by kvnkho** [comment]
Thanks!
